opensquirrel/exporter/quantify_scheduler_exporter.py

```diff
@@ -1,20 +1,29 @@
 from __future__ import annotations
 
 import math
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
-from opensquirrel.circuit import Circuit
 from opensquirrel.common import ATOL
 from opensquirrel.default_gates import X, Z
 from opensquirrel.exceptions import ExporterError, UnsupportedGateError
-from opensquirrel.ir import BlochSphereRotation, ControlledGate, IRVisitor, MatrixGate, Measure, Qubit
+from opensquirrel.ir import (
+    BlochSphereRotation,
+    ControlledGate,
+    IRVisitor,
+    MatrixGate,
+    Measure,
+    Qubit,
+)
 
 try:
     import quantify_scheduler
     import quantify_scheduler.operations.gate_library as quantify_scheduler_gates
 except ModuleNotFoundError:
     pass
 
+if TYPE_CHECKING:
+    from opensquirrel.circuit import Circuit
+
 
 # Radian to degree conversion outcome precision
 DEG_PRECISION = 5
```

Comment:
For consistency, we may want to rewrite `DEG_PRECISION` to `FIXED_DEG_PRECISION`

Reply:
Done